trio/tests/test_exports.py

```diff
@@ -96,6 +97,31 @@ def no_underscores(symbols):
         script = jedi.Script(f"import {modname}; {modname}.")
         completions = script.complete()
         static_names = no_underscores(c.name for c in completions)
+
+    elif tool == "pyright_verifytypes":
+        # it's maybe possible to run this through pyright internals
```

Comment:
FYI, pyright is written in JS so we won't really be able to poke at its internals. The pyright python package is just a transparent wrapper around setting up a tiny node env and using it.